proto/mixer.proto

```diff
@@ -817,6 +829,14 @@ service Mixer {
     };
   }
 
+  // Get landing page info for a place.
+  rpc GetLandingPage(GetLandingPageRequest) returns (GetLandingPageResponse) {
+    option (google.api.http) = {
+      post: "/node/landing-page"
```

Comment:
can you also add statsvar as paramters? So client can ask only data for certain stats vars

Reply:
Done.